packages/ui-react/src/components/ChooseOfferForm/ChooseOfferForm.tsx

```diff
@@ -69,8 +69,8 @@ const ChooseOfferForm: React.FC<Props> = ({
     return null;
   };
 
-  const OfferBox: OfferBox = ({ offer, title, ariaLabel, secondBenefit, periodString }) => (
-    <div className={styles.offer}>
+  const OfferBox: OfferBox = ({ offer, title, secondBenefit, periodString }) => (
+    <div className={styles.offer} role="option">
```

Comment:
I'm not sure this is needed. We use the `input[type="radio"]` element, which already has the correct role and keyboard interaction. The `div` only represents the visual appearance of the native input.

What do you think? @langemike @MelissaDTH

Reply:
I think having all this HTML (eg. h2, ul, hr) in a <label> is not good and can lead to unexpected screen reader behaviour. This is my assumption. Also this is not valid HTML. So that could at least be something that be notied.

By providing a `role="option"` we make the descendants presentational. Reading the full contents in one sentence. Also not the ideal solution, but at least better than was IMHO.

Also I was hoping to make it more accessible for keyboard and screen reader navigation (toggeling/switching between the two). But in both situations (old vs. new) I'm unable to do so. This could be me. But using the arrow-keys or tab-key . I know the focus-outline is missing. But i definitly need guidance how to use this 😓